lib/ansible/modules/crypto/openssl_privatekey.py

```diff
@@ -325,6 +352,8 @@ def dump(self):
             'changed': self.changed,
             'fingerprint': self.fingerprint,
         }
+        if self.backup_path:
```

Comment:
Should this field be present in check mode if the key would be regenerated?

Reply:
Good question. I'll take a look what `copy`/`template` do.

Reply:
Turns out `copy` does not return it in check mode, and `template` doesn't return anything (neither in check mode nor in normal mode) except the standard `changed` result. So I guess we don't need to return it for check mode.

Reply:
... or we need to file issues against these 2 modules. ;-)

I guess it's fine either way though, so let's keep it similar to `copy` for now.